shared-data/protocol/schemas/6.json

```
@@ -846,74 +846,66 @@
           },
 
           {
-            "description": "Heater Shaker Module Set Target Shake Speed: Module will begin accelerating to the target shake speed. This command is non-blocking, it does not delay protocol execution while approaching the target shake speed.",
+            "description": "Heater Shaker Module Set Target Temperature: Module will delay protocol execution until the target specified target temperature is reached. This command is locking, it delays protocol execution while approaching the target temperature.",
             "type": "object",
             "required": ["id", "commandType", "params"],
             "properties": {
               "id": { "type": "string" },
               "commandType": {
-                "enum": ["heaterShakerModule/setTargetShakeSpeed"]
+                "enum": ["heaterShakerModule/setTargetTemperature"]
               },
               "params": {
-                "required": ["moduleId", "rpm"],
+                "required": ["moduleId", "temperature"],
                 "properties": {
                   "moduleId": {
                     "type": "string",
                     "description": "Unique identifier of Heater Shaker Module to target. Must be a key from the top level 'modules' object"
                   },
-                  "rpm": {
+                  "temperature": {
                     "type": "number",
-                    "description": "target orbital rotations per minute"
+                    "description": "target temperature to heat to in °C"
                   }
                 }
               }
             }
           },
 
           {
-            "description": "Heater Shaker Module Await Temperature: Delay protocol execution until the specified target temperature is reached.",
+            "description": "Heater Shaker Module Set Target Shake Speed: Module will delay protocol execution until the target shake speed is reached. This command is blocking, it delays protocol execution while approaching the target shake speed.",
```

I prefer the previous description of `it delays protocol execution until the specified target speed has reached`. It has a clear ending condition for the delay. What was the reason for this change?

---

the github diff is a little confusing here, i removed the `heaterShakerModule/awaitTargetShakeSpeed` command since the H/S ramps shake speed so quickly

---

I think this description needs more about what the command will do and less about "blocking," which is strange terminology to expose to users IMO. I think something like this is a little more clear

```suggestion
            "description": "Heater Shaker Module Set Target Shake Speed: Start the module's shake procedure. The protocol will proceed once the target shake speed is reached.",
```